lightblue-client-integration-test/src/main/java/com/redhat/lightblue/client/integration/test/LightblueExternalResource.java

```diff
@@ -48,7 +48,7 @@ protected AbstractLightblueClientCRUDController getControllerInstance() {
             try {
                 if (removeHooks == Boolean.TRUE)
```

Comment:
removeHooks is not Boolean, it is boolean. So if(removeHooks) { .. } else { ... } is sufficient